projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy_optimise.php

```diff
@@ -421,7 +422,7 @@ public function compress_numbers( $subvalue ) {
 	 */
 	public function analyse_css_number( $string ) {
 		// most simple checks first
-		if ( strlen( $string ) == 0 || ctype_alpha( $string[0] ) ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
+		if ( strlen( $string ) === 0 || ctype_alpha( $string[0] ) ) {
```

Comment:
Now that I think about it, this could be simplified even further.
```suggestion
		if ( $string === '' || ctype_alpha( $string[0] ) ) {
```